---
linktitle: "Publications"
title: "Publications"
date: 2020-04-29T23:13:53+02:00
draft: true
---


Nakamura, K., Saredi, G., Becker, J., Foster, B., Nguyen, N., Beyer, T., Cesa, L., Faull, P., **Lukauskas, S.**, Frimurer, T., Chapman, J., Bartke, T., Groth, A. (2019). H4K20me0 recognition by BRCA1â€“BARD1 directs homologous recombination to sister chromatids Nature Cell Biology  21(3), 311-318. https://dx.doi.org/10.1038/s41556-019-0282-9

Kalesh, K., **Lukauskas, S.**, Borg, A., Snijders, A., Ayyappan, V., Leung, A., Haskard, D., DiMaggio, P. (2019). An Integrated Chemical Proteomics Approach for Quantitative Profiling of Intracellular ADP-Ribosylation Scientific Reports  9(1), 6655. https://dx.doi.org/10.1038/s41598-019-43154-1

\*Fan, S., \*Geissmann, Q., \*Lakatos, E., **\*Lukauskas, S.**, Ale, A., Babtie, A., Kirk, P., Stumpf, M. (2016). MEANS: python package for Moment Expansion Approximation, iNference and Simulation Bioinformatics  32(18), 2863-2865. https://dx.doi.org/10.1093/bioinformatics/btw229

**Lukauskas, S.**, Visintainer, R., Sanguinetti, G., Schweikert, G. (2016). DGW: an exploratory data analysis tool for clustering and visualisation of epigenomic marks BMC Bioinformatics  17(Suppl 16), 447. https://dx.doi.org/10.1186/s12859-016-1306-0

\* Joint first authorship
